suite-native/module-accounts-import/src/components/AccountImportHeader.tsx

```diff
@@ -32,18 +34,33 @@ type NavigationProp = StackToTabCompositeProps<
     RootStackParamList
 >;
 
-export const AccountImportHeader = ({ activeStep }: AccountImportHeaderProps) => {
+export const AccountImportHeader = ({ activeStep, origin }: AccountImportHeaderProps) => {
     const { applyStyle } = useNativeStyles();
     const navigation = useNavigation<NavigationProp>();
     const isOnboardingFinished = useSelector(selectIsOnboardingFinished);
 
-    const handleCloseOnboarding = () =>
-        navigation.navigate(RootStackRoutes.AppTabs, {
-            screen: AppTabsRoutes.HomeStack,
-            params: {
-                screen: HomeStackRoutes.Home,
-            },
-        });
+    const handleCloseOnboarding = () => {
+        switch (origin) {
+            case HomeStackRoutes.Home:
+                navigation.navigate(RootStackRoutes.AppTabs, {
+                    screen: AppTabsRoutes.HomeStack,
+                    params: {
+                        screen: HomeStackRoutes.Home,
```

Comment:
What exactly is this `screen` param? I can't see it in `navigation/src/navigators.ts`